### PR TITLE
Refactor: Create uninstall submodule

### PR DESCRIPTION
**Pull request by k-nasa**
Since 'uninstall' was found in the 'install' module, it split.
And I moved duplicate functions to utils.

### CODE REVIEW

File: src/cargo/ops/mod.rs [hunk 1]
```diff
@@ -39,7 +40,9 @@ mod cargo_pkgid;
 mod cargo_read_manifest;
 mod cargo_run;
 mod cargo_test;
+mod cargo_uninstall;
 mod fix;
 mod lockfile;
 mod registry;
 mod resolve;
+mod utils;
```
[the remark this answers is not in the document]
Comment (k-nasa) [reply src/cargo/ops/mod.rs]: Okay. I will have time to fix it.

File: src/cargo/ops/cargo_uninstall.rs [hunk 2]
```diff
@@ -0,0 +1,156 @@
...
+fn uninstall_pkgid(
+    crate_metadata: &FileLock,
+    mut metadata: CrateListingV1,
+    pkgid: PackageId,
+    bins: &[String],
+    config: &Config,
+) -> CargoResult<()> {
+    let mut to_remove = Vec::new();
+    {
+        let mut installed = match metadata.v1.entry(pkgid) {
```
Comment (k-nasa) [inline src/cargo/ops/cargo_uninstall.rs]: @dwijnand  I needed to refer to v1 here, so I changed the field to public.
I am not familiar with Rust so much, so please tell me if there is a better way.

Comment (dwijnand) [reply src/cargo/ops/cargo_uninstall.rs]: Add a `v1()` "getter" method to metadata, that simply returns the `v1` field.

File: src/cargo/ops/mod.rs (hunk 1 above)
[the remark this answers is not in the document]
Comment (k-nasa) [reply src/cargo/ops/mod.rs]: ops::utils -> ops::common_for_install_and_uninstall
Somewhat verbose name, but how about this?

**Comment by k-nasa** [comment]
@dwijnand As I fixed it, would you see it again when there is time❓

**Comment by dwijnand** [comment]
I'm really enjoying your "common between install and uninstall" module name! :D. It's specific enough to avoid being a catch-all module, but it's long enough that I expect it'll find a nicer name sooner or later (hopefully not utils!).

So it's a good enough working title to land this, for me.  Thank you for the PR.

@bors: r+

File: src/cargo/ops/cargo_uninstall.rs (hunk 2 above)
Comment (k-nasa) [reply src/cargo/ops/cargo_uninstall.rs]: I see. thank you for answering! 🎉

**Pull request merged**